src/main/java/se/liu/ida/hefquin/engine/federation/access/DataRetrievalInterface.java

```diff
@@ -9,4 +9,6 @@ public interface DataRetrievalInterface
 	boolean supportsSPARQLPatternRequests();
 
 	boolean supportsRequest( final DataRetrievalRequest req );
+	
+	int getID();
```

Comment:
Please add a JavaDoc comment for this function (see the one in LogicalOperator).